### PR TITLE
Added clear cache support

### PR DESCRIPTION
**Pull request by DigneshGujarathi**
## Test Plan

Testing is needed by upgrading the version and build.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
     "eslint-plugin-react-hooks": "4.2.0",
     "formik": "^2.2.6",
     "formik-material-ui": "^3.0.1",
+    "global": "^4.4.0",
```
Comment (mdshamoon) [inline package.json:67]: Do we need this?

File: package.json [hunk 2]
```diff
@@ -88,6 +90,7 @@
     "react-string-replace": "^0.4.4",
     "react-tiny-link": "^3.6.1",
     "react-viewer": "^3.2.2",
+    "serve": "^11.3.2",
```
Comment (mdshamoon) [inline package.json:93]: Do we need this?

File: src/App.tsx [hunk 3]
```diff
@@ -52,7 +59,9 @@ const App = () => {
     <SessionContext.Provider value={values}>
       <ApolloProvider client={gqlClient(history)}>
         <ErrorHandler />
-        <Suspense fallback={Loading}>{routes}</Suspense>
+        <ClearCacheProvider duration={60000}>
```
Comment (mdshamoon) [inline src/App.tsx:62]: Update this to 1 hour.

Comment (DigneshGujarathi) [reply src/App.tsx]: Yes, updated to 360000

File: package.json (hunk 2 above)
Comment (DigneshGujarathi) [reply package.json]: No, updated the PR

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**